test/models/friend_request_test.rb

```diff
@@ -3,11 +3,24 @@
 class FriendRequestTest < ActiveSupport::TestCase
 
   def setup
-    @friend_request = friend_requests(:one)
+    @user = users(:one)
+    @other_user = users(:two)
   end
 
-  test 'should valid' do
-    assert @friend_request.valid?
+  test 'stastus should be valid' do
+    # presence
+    @request = @user.friend_requests_sent.build(receiver: @other_user)
+    assert_not @request.valid?
+    assert @request.errors[:status].any?
+    # inclusion
+    @request.status = 5
```

Comment:
There must be a simpler way to test the validity of a friendrequest that is more related to how the app operates. For instance, lines `12-14` probably have made the case for an invalid request. Why did we need to have a test from the request.status?

We can simply use the fixtures to create two kinds of friend requests - one valid (probably missing one required attribute) and one valid. Then we can just make our assertions from them.